app-sso/service-operators/identity-providers.md

```diff
@@ -103,24 +101,21 @@ spec:
   identityProviders:
     - name: ldap
       ldap:
-        server:
-          scheme: ldap
-          host: my-ldap.com
-          port: 389
-          base: ""
+        url: "ldaps://example.com:636"
         bind:
-          dn: uid=binduser,ou=Users,o=5d03d6ac6eed091436a8d664,dc=jumpcloud,dc=com
+          dn: uid=binduser,ou=Users,dc=example,dc=com
           passwordRef:
             name: ldap-password
         user:
-          searchFilter: uid={0}
-          searchBase: ou=Users,o=5d03d6ac6eed091436a8d664,dc=jumpcloud,dc=com
+          searchBase: ou=Users,dc=example,dc=com
+          searchFilter: cn={0}
         group:
-          searchFilter: member={0}
-          searchBase: ou=Users,o=5d03d6ac6eed091436a8d664,dc=jumpcloud,dc=com
-          searchSubTree: true
-          searchDepth: 10
-          roleAttribute: cn
+          search:
+            filter: member={0}
+            base: ou=Users,dc=example,dc=com
```

Comment:
@Kehrlann do we want to refer to this as memberOf to make it look more AD-ish?

Reply:
@candicenonsense Ah sorry, I missed this comment somehow.

`memberOf` is only used when `group.search` is turned off. For clarification, `memberOf` is an attribute of the user object, while `member` is an attribute of the group object.